<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindNotification</class>
 <widget class="QWidget" name="FindNotification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="Pattern_"/>
   </item>
   <item>
    <widget class="QCheckBox" name="SearchBackwards_">
     <property name="text">
      <string>Backwards</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="MatchCase_">
     <property name="text">
      <string>Case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="WrapAround_">
     <property name="text">
      <string>Cycle</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FindButton_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
     <property name="icon">
      <iconset theme="edit-find"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CloseButton_">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="window-close"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Pattern_</sender>
   <signal>returnPressed()</signal>
   <receiver>FindButton_</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
